contentcuration/contentcuration/models.py

```diff
@@ -409,6 +409,17 @@ class ContentNode(MPTTModel, models.Model):
 
     objects = TreeManager()
 
+    def __init__(self, *args, **kwargs):
+        super(ContentNode, self).__init__(*args, **kwargs)
+        self._original_fields = self._as_dict()
```

Comment:
Why do we need to store the original fields as a dict now? Can you add a comment as to why?